package.json

```diff
@@ -50,6 +51,7 @@
     "pegjs-loader": "^0.5.1",
     "postcss-loader": "^1.3.3",
     "raw-loader": "^0.5.1",
+    "react-test-renderer": "^15.5.4",
```

Comment:
Enzyme won't work without this since we are on the latest version of react.  As we bump our react version we will need to bump this as well to match, one of the reason potentially testing component rendering will be more of a hassle than a positive.